MediaBrowser.Providers/Plugins/TheTvdb/TvdbSeriesProvider.cs

```diff
@@ -239,9 +232,9 @@ private async Task<List<RemoteSearchResult>> FindSeriesInternal(string name, str
             {
                 var tvdbTitles = new List<string>
                 {
-                    GetComparableName(seriesSearchResult.SeriesName)
+                    seriesSearchResult.SeriesName
```

Comment:
Should `comparableName` also be added `tvdbTitles`?

Reply:
That was one of the reasons search did not work properly. `GetCombarableName` sometimes drops too much information from the title causing sort at the end of `FindSeriesInternal` to place bad results on top of the list. By adding only full TVDB titles to this list we can sort it by:
1. Contains exact search name
2. Contains parsed search name
3. Exact production year (if present)
4. And only now `comparableName`